runtime/compiler/trj9/codegen/J9CodeGenerator.hpp

```diff
@@ -382,7 +386,8 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGeneratorConnector
    enum // Flags
       {
       HasFixedFrameC_CallingConvention = 0x00000001,
-      SupportsMaxPrecisionMilliTime    = 0x00000002
+      SupportsMaxPrecisionMilliTime    = 0x00000002,
+      SupportsSIMDStringCaseConv       = 0x00000004
```

Comment:
Can we rename this to something more generic. You don't have to use SIMD here. You can also "vectorize" the operation in general purpose registers. Could we rename to perhaps `SupportsInlineStringCaseConversion` and add a Doxygen comment on what exactly this flag does?

We should then appropriately rename the getter and setter a few lines above.